src/functions/write_detrend_data.py

```diff
@@ -0,0 +1,51 @@
+import os
+import sys
+
+import pandas as pd
+
+sys.path.append("./src/functions")
+from detrend import LinearMA  # noqa: E402
+
+# SPECIFY DETREND MODEL AND DIRECTORY IN WHICH SAVE PROCESSED DATA
+
+detrend_model = LinearMA(window=200)
+model_name: str = "LinearMA"
+model_options: str = "window-201"  # format: option1-value_option2-value
+processed_data_folder = f"data/processed_data/detrend_data/{model_name}/{model_options}"
+
+
+# GATHER FILES TO PROCESS
+
+data_folder = "data/raw_data"
+filenames = [f for f in os.listdir(data_folder) if f.endswith(".csv")]
+
+# READ FILES
+
+data_files: dict = {
+    filename: pd.read_csv(
+        f"{data_folder}/{filename}", parse_dates=["Date"], index_col="Date"
+    )
+    for filename in filenames
+}
+
+# DETREND FILES
```

Comment:
nit: no uppercase, write in lower case (or lowercase with first letter uppercased)

Reply:
done